mne/io/edf/tests/test_edf.py

```diff
@@ -538,6 +538,17 @@ def test_read_latin1_annotations(tmp_path):
     annot_file = tmp_path / "annotations.txt"
     with open(annot_file, "wb") as f:
         f.write(annot)
+
+    # Test reading directly from file
+    onset, duration, description = _read_annotations_edf(
```

Comment:
```suggestion
   onset, duration, description, ch_names = _read_annotations_edf(
```
following https://github.com/mne-tools/mne-python/pull/11960